nixos/modules/programs/dwm-status.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.dwm-status;
+
+  order = concatMapStringsSep "," (feature: ''"${feature}"'') cfg.order;
+
+  configFile = pkgs.writeText "dwm-status.toml" ''
+    order = [${order}]
+
+    ${cfg.extraConfig}
+  '';
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    programs.dwm-status = {
```

Comment:
I think this should be under `services`, not `programs`.

Reply:
Is there anywhere a definition for the difference between service and program?

Reply:
It's just that `services.*` is reserved for things that start systemd services, while `programs.*` is mostly for config files for programs when you run them yourself.

Reply:
Seems reasonable, moved the module.